cmd/server/config/base.yaml

```diff
@@ -0,0 +1,138 @@
+persistence:
```

Comment:
We should not a config here. Config files should live under config directory.

Reply:
deleted